src/libstd/sys/windows/ext/fs.rs

```diff
@@ -18,6 +18,43 @@ use path::Path;
 use sys;
 use sys_common::{AsInnerMut, AsInner};
 
+/// Windows-specific extensions to `File`
+#[unstable(feature = "file_offset", issue = "35918")]
+pub trait FileExt {
+    /// Seeks to a given position and reads a number of bytes.
+    ///
+    /// The offset is relative to the start of the file and thus independent
+    /// from the current cursor. The current cursor **is** affected by this
+    /// function.
```

Comment:
Could you clarify that the current cursor is set to the end of the read when the function returns? Also, could you clarify what happens to the cursor if an error happens?

Also just as a sanity check, if `Ok(n)` is returned then the cursor is `offset + n`, right?